sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/TPCDSQueryBenchmarkArguments.scala

```diff
@@ -32,6 +36,10 @@ class TPCDSQueryBenchmarkArguments(val args: Array[String]) {
           dataLocation = value
           args = tail
 
+        case ("--query-filter") :: value :: tail =>
+          queryFilter = value.toLowerCase(Locale.ROOT).split(",").map(_.trim).toSet
```

Comment:
Could you also make `"--data-location"` case insensitive?

Reply:
ok